### ISSUE TITLE
Add thedfirreport.com

### ISSUE THREAD

**Issue by edoardottt** [edoardottt/awesome-hacker-search-engines#39]
Add https://thedfirreport.com/

**Comment by aybeedee** [comment]
Will it come under the "Attack Surface" heading?

**Comment by edoardottt** [comment]
No, under [`Threat Intelligence`](https://github.com/edoardottt/awesome-hacker-search-engines#threat-intelligence).

The description should be `Real Intrusions by Real Attackers, The Truth Behind the Intrusion`.
